usaon_vta_survey/__init__.py

```diff
@@ -81,7 +85,11 @@ def before_request():
     app.register_blueprint(login_bp)
     app.register_blueprint(logout_bp)
     app.register_blueprint(blueprint, url_prefix="/google_oauth")
-    app.register_blueprint(response.bp)
+    app.register_blueprint(bp)
```

Comment:
@rmarow `bp` here is hard to read. What about `response_bp`?

I feel `obs_bp` and `dp_bp` would also be more readable spelling out the words.